command/snapshot/inspect/snapshot_inspect_test.go

```diff
@@ -110,3 +133,48 @@ func TestSnapshotInspectCommand(t *testing.T) {
 		}
 	}
 }
+
+func TestSnapshotInspectEnhanceCommand(t *testing.T) {
+	t.Parallel()
+	a := agent.NewTestAgent(t, ``)
+	defer a.Shutdown()
+	client := a.Client()
+
+	dir := testutil.TempDir(t, "snapshot")
+	defer os.RemoveAll(dir)
+
+	file := filepath.Join(dir, "backup.tgz")
+
+	// Save a snapshot of the current Consul state
+	f, err := os.Create(file)
+	if err != nil {
+		t.Fatalf("err: %v", err)
+	}
+
+	snap, _, err := client.Snapshot().Save(nil)
+	if err != nil {
+		f.Close()
+		t.Fatalf("err: %v", err)
+	}
+	if _, err := io.Copy(f, snap); err != nil {
+		f.Close()
+		t.Fatalf("err: %v", err)
+	}
+	if err := f.Close(); err != nil {
+		t.Fatalf("err: %v", err)
+	}
+
+	// Inspect the snapshot
+	ui := cli.NewMockUi()
+	c := New(ui)
+	args := []string{"-enhance", file}
+
+	code := c.Run(args)
+	if code != 0 {
+		t.Fatalf("bad: %d. %#v", code, ui.ErrorWriter.String())
+	}
+	// Set golden file to update
+	*update = true
```

Comment:
Oh! You don't need to do this, this is why the test is passing.

If you want to update the value in the file, run the test with the `-update` flag:

```
go test ./command/snapshot/inspect
```